### DISCUSSION TITLE
Stitch works not correctly

### DISCUSSION THREAD

**Discussion by phamkhactu** [OpenStitching/stitching#154] [Q&A]
Thanks for amazing your work!!

From your response for [issue](https://github.com/OpenStitching/stitching/issues/54#issuecomment-1451578143) and tutorial. I've created my code. All my images are sort orders.

```
from stitching import Stitcher
from stitching.images import Images
import cv2

stitcher = Stitcher()
images= [
    'imgs/0.jpg',
    'imgs/1.jpg',
    'imgs/2.jpg',
    'imgs/3.jpg',
    'imgs/4.jpg',
    'imgs/5.jpg'
]
panorama = stitcher.stitch(images) 
cv2.imwrite("panorama.jpg", panorama)
```
But I save panorama image, it stitch not well. I see that ```panorama``` is the same with ```1.jpg``` not combine all from ```0.jpg``` to ```5.jpg```
 
![panorama](https://github.com/OpenStitching/stitching/assets/42369268/ae1223b8-8c82-43bc-a88f-fbdde965ed36)

Here is my [images](https://drive.google.com/drive/folders/1ud84Ijon4lqho-w3QI5wOwthmqZWLbZN?usp=sharing)

**Comment by lukasalexanderweber** [comment]
Try `stitcher.stitch_verbose` for Debugging. Do you get a warning message?

**Reply by phamkhactu** [reply]
Hi  @lukasalexanderweber  
yes, I get it:

```
StitchingWarning: Not all images are included in the final panorama.
                          If this is not intended, use the 'matches_graph_dot_file'
                          parameter to analyze your matches. You might want to
                          lower the 'confidence_threshold' or try another 'detector'
                          to include all your images.

```

if i set: ```stitcher = Stitcher(confidence_threshold=0, range_width=1)```
I get error: 
```
    panorama = stitcher.stitch(images) 
  File "/media/tupk/DATA/anaconda3/envs/dl/lib/python3.8/site-packages/stitching/stitcher.py", line 102, in stitch
    cameras = self.refine_camera_parameters(features, matches, cameras)
  File "/media/tupk/DATA/anaconda3/envs/dl/lib/python3.8/site-packages/stitching/stitcher.py", line 157, in refine_camera_parameters
    return self.camera_adjuster.adjust(features, matches, cameras)
  File "/media/tupk/DATA/anaconda3/envs/dl/lib/python3.8/site-packages/stitching/camera_adjuster.py", line 48, in adjust
    raise StitchingError("Camera parameters adjusting failed.")
stitching.stitching_error.StitchingError: Camera parameters adjusting failed.
```

**Reply by lukasalexanderweber** [reply]
You could try another adjuster

**Reply by phamkhactu** [reply]
I'm a newbie in this. I'm glad if you give me some code for solving it.
Thank you very much!
